financialaid/exceptions.py

```diff
@@ -0,0 +1,10 @@
+"""
+Exceptions for financialaid.
+"""
+
+
+class NotSupportedException(Exception):
+    """
+    Not supported by current financial aid system.
+    """
+    pass
```

Comment:
This is not necessary since you have the docstring. Can you remove this line please?